### DISCUSSION TITLE
HTTP QUERY method

### DISCUSSION THREAD

**Discussion by 0xflotus** [curl/curl#10281] [Q&A] [answered]
Will or already do `curl` support the [new QUERY method](https://www.ietf.org/archive/id/draft-ietf-httpbis-safe-method-w-body-02.html)?

**Comment by bagder** [comment, marked as answer]
You can send QUERY methods like this: `curl -d "send this" -X QUERY`

We might want to give it a think if we should support it even better than this going forward.
